src/librustdoc/clean/types.rs

```diff
@@ -640,6 +667,8 @@ crate enum ItemKind {
     ExternCrateItem {
         /// The crate's name, *not* the name it's imported as.
         src: Option<Symbol>,
+        /// The id of the `extern crate` statement, not the crate itself.
+        crate_stmt_id: LocalDefId,
```

Comment:
Why can't this use `Item.inline_stmt_id`? Or I guess is this not quite the same as `inline_stmt_id`? Still, it seems like they should be merged somehow since they're similar.